Tasks/AzureRmWebAppDeployment/VariableSustituter.ps1

```diff
@@ -0,0 +1,51 @@
+﻿Param(
+  [Parameter(mandatory=$true)][string]$webAppFolderPath,
+  [Parameter(mandatory=$true)][string]$ConfigFileRegex
+)
+
+# define mime supported mime type
+$ApplicationXMLMimeType = "application/xml" 
+
+function Get-MimeType()
+{
+    param($extension = $null);
+ 
+    $mimeType = $null;
+    
+    if ( $null -ne $extension )
```

Comment:
Use -not String::IsNullOrEmpty($param) which is recommended in powershell for null checks